rllib/agents/ppo/ppo.py

```diff
@@ -239,7 +239,7 @@ def __init__(self):
     @Deprecated(
         old="ray.rllib.agents.ppo.ppo.DEFAULT_CONFIG",
         new="ray.rllib.agents.ppo.ppo.PPOConfig(...)",
-        error=False,
+        error=True,
```

Comment:
Are we moving to PPOConfig **now**? What about other configs? Feels kind of weird hard decprecating the PPO config on 1.13.

Reply:
I guess it's just too tedious to keep both for now, eh?

Reply:
Oh yeah, good catch!